tests/console/java.pm

```diff
@@ -58,7 +58,7 @@ sub run {
         else {
             diag "There are no java bootstrap packages";
             print "There are no java bootstrap packages\n";
-            zypper_call("in java-*", exitcode => [0, 107]);
+            zypper_call "in java-*";
```

Comment:
can you return brackets back ? :)

Reply:
The brackets were not there initially. The same zypper call before this one does not have them.

Reply:
this pool is not real revert of #7184 anyway because you skipping part with `evolution_prepare_servers.pm` so nothing force you to be so explicit

Reply:
Fair enough. Updated both of them.